epoxy-processor/src/main/java/com/airbnb/epoxy/AttributeInfo.java

```diff
@@ -142,6 +155,43 @@ private boolean isFieldPackagePrivate(Element attribute) {
   }
 
   /**
+   * Checks if the given private field has getter and setter for access to it
+   */
+  private void findGetterAndSetterForPrivateField(ErrorLogger errorLogger) {
```

Comment:
To simplify, what if we assumed/enforced that the getter/setter have the same name as the field. Our generated getter/setter will already share that name so it seems like it would be a good thing for consistency anyway. Does this make sense in kotlin?

I previously made a `ProcessorUtilsgetMethodOnClass` method that you may be able to reuse to simplify this, although you may want to add some modifier checking into that method (to check !static, etc).